Autopilot/PathManager/Inc/pathModeSelector.hpp

```diff
@@ -165,6 +185,12 @@ class PathModeSelector {
     private: 
         PathModeSelector();
         static PathModeSelector* singleton;
+        
+        /*
+         * Function to enqueue a new instruction onto the instruction queue.
+         */
+        void enqueueInstruction(Telemetry_PIGO_t newInstruction);
+        instructionQueueNode* first_instr;
```

Comment:
In your comment for the function here I'd write an @param field to explain what the parameter is.

Also, add a comment to the instructionQueueNode pointer so it's more clear what it is being used for. Maybe renaming it to `cruising_edit_path_first_instr` may be more clear (especially if we add more queues in the future)